README.md

```diff
@@ -1,43 +1,51 @@
-homebrew-neovim
-===============
+# homebrew-neovim
+
 [![Build Status](https://travis-ci.org/neovim/homebrew-neovim.svg?branch=master)](https://travis-ci.org/neovim/homebrew-neovim)
 
-Homebrew formula for neovim.
+Homebrew formula for Neovim.
 
-## Install (stable)
+## Installation
 
-To install the latest stable release:
+Since this formula isn't part of Homebrew, add it as tap first:
 
-    $ brew install neovim/neovim/neovim
+    $ brew tap neovim/neovim
 
-To **upgrade** from a previous version:
+To install the latest stable release:
```

Comment:
upgrade was bolded, and near the top, because people were missing it and stuck on old versions and then reporting old bugs...